Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.py

```diff
@@ -1149,15 +1146,21 @@ def fetch_incidents(client: Client,
 def main():
     os.environ['PAN_CREDENTIALS_DBFILE'] = os.path.join(gettempdir(), 'pancloud_credentials.json')
     params = demisto.params()
-    registration_id_and_url = params.get(REGISTRATION_ID_CONST).split('@')
+    registration_id_and_url = (params.get('credentials_id', {}).get('identifier') or params.get('reg_id')).split('@')
+    if not registration_id_and_url:
+        raise DemistoException('ID must be provided.')
     if len(registration_id_and_url) != 2:
         token_retrieval_url = "https://oproxy.demisto.ninja"  # guardrails-disable-line
     else:
         token_retrieval_url = registration_id_and_url[1]
     registration_id = registration_id_and_url[0]
     # If there's a stored token in integration context, it's newer than current
-    refresh_token = demisto.getIntegrationContext().get(REFRESH_TOKEN_CONST) or params.get(REFRESH_TOKEN_CONST)
-    enc_key = params.get(ENCRYPTION_KEY_CONST)
+    refresh_token = params.get('credentials_refresh_token', {}).get('password') or params.get('refresh_token')
+    if not refresh_token:
+        raise DemistoException('Token must be provided.')
+    enc_key = params.get('credentials_id', {}).get('password') or params.get('auth_key')
+    if not enc_key:
+        raise DemistoException('ID must be provided.')
```

Comment:
```suggestion
        raise DemistoException('ID must be provided.')
```
Here in the exception is should be `Key` no?